lib/blueprinter/view_collection.rb

```diff
@@ -49,15 +51,15 @@ def identifier_fields
     # @return [Array<(Hash, Hash<String, NilClass>)>] fields, excluded_fields
     def sortable_fields(view_name)
       excluded_fields = {}
-      fields = views[:default].fields
+      fields = views[:default].fields.clone
       views[view_name].included_view_names.each do |included_view_name|
         next if view_name == included_view_name
 
         view_fields, view_excluded_fields = sortable_fields(included_view_name)
-        fields = merge_fields(fields, view_fields)
+        fields.merge!(view_fields)
         excluded_fields.merge!(view_excluded_fields)
       end
-      fields = merge_fields(fields, views[view_name].fields)
+      fields.merge!(views[view_name].fields) unless view_name == :default
```

Comment:
```suggestion
      fields = views[:default].fields
      views[view_name].included_view_names.each do |included_view_name|
        next if view_name == included_view_name

        view_fields, view_excluded_fields = sortable_fields(included_view_name)
        fields.merge(view_fields) # create a new hash for manipulation
        excluded_fields.merge!(view_excluded_fields)
      end
      fields.merge!(views[view_name].fields)
      fields.merge!(views[view_name].fields) unless view_name == :default
```
can we use `merge` once to create a new hash and then use `merge!` to manipulate it further?

Reply:
This will create n hashes, where n is the number of included views (which is expensive because `Hash#merge` is just an alias for `hash.dup.merge!(...)`), while the clone+merge! method creates exactly one hash per view. I'm happy to rewrite the code this way if this is more readable.

Reply:
fair enough, I overlooked the `views[view_name].included_view_names.each` iteration when optimising for readability + performance! 😅 This LGTM.